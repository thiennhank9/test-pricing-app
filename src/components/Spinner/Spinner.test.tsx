import TestRenderer from "react-test-renderer";
import Spinner from "./Spinner";

describe("Spinner", () => {
  test("render spinner", () => {
    const testRenderer = TestRenderer.create(<Spinner />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
