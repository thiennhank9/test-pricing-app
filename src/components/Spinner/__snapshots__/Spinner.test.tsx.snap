// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner render spinner 1`] = `
<div
  className="spinner"
/>
`;
